osu.Game.Rulesets.Catch.Tests/CatchBeatmapConversionTest.cs

```diff
@@ -14,7 +14,6 @@
 namespace osu.Game.Rulesets.Catch.Tests
 {
     [TestFixture]
-    [Timeout(10000)]
```

Comment:
To clarify, I presume that the base `BeatmapConversionTest` is taking over the onus of performing the 10s timeout, right?

Reply:
Yeah, I moved it there.